bootstrap/common.sh

```diff
@@ -31,7 +31,7 @@ verify_prereqs() {
 check_terraform_version() {
   local IFS='.'
   local current_version_string="${2:-$( terraform --version | awk 'NR==1 {print $2}' )}"
-  local requirement_version_string=${1:-0.6.12}
+  local requirement_version_string=${1:-0.6.10} # This is to support Rackspace. Access IPs are not properly exported by the Terraform Openstack provider in versions > 0.6.10. See https://github.com/hashicorp/terraform/issues/5358
```

Comment:
Changed it back to 0.6.12. I'm using the Openstack provider built from source with the fix implemented though, so I would wait for TF 0.6.13 before merging.